forms/src/test/java/org/odk/collect/forms/instances/InstanceTest.kt

```diff
@@ -0,0 +1,14 @@
+package org.odk.collect.forms.instances
+
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.junit.Test
+
+class InstanceTest {
+
+    @Test
+    fun `canDeleteBeforeSend is true by default`() {
```

Comment:
> I think it would be good to have a test in the same class where we just save a form as a draft and test if it can be deleted.

I'd thought about that (I actually wrote that test then deleted it), but I realized we already had a test for finalized forms and felt like adding this unit test was simpler than testing all the possible states that saved forms could be in.